### PR TITLE
Adds a unit test for dupe planes

### PR DESCRIPTION
**Pull request by Time-Green**
Woohoo!
Merge #70124 first, or everyones unit tests will break

Do I even add a changelog? meh

### CODE REVIEW

File: code/modules/unit_tests/plane_dupe_detector.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+///Checks if some idiot didnt give two planes the same layer
```
Comment (Mothblocks) [inline code/modules/unit_tests/plane_dupe_detector.dm:1]: cmon

File: code/modules/unit_tests/plane_dupe_detector.dm [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+/datum/unit_test/plane_dupe_detector
+
+/datum/unit_test/plane_dupe_detector/Run()
+	var/list/plane_integer_list = list()
+	for(var/atom/movable/screen/plane_master/air as anything in subtypesof(/atom/movable/screen/plane_master))
```
Comment (Mothblocks) [inline code/modules/unit_tests/plane_dupe_detector.dm:6]: air? just name this plane_master_path or something

Comment (Time-Green) [reply code/modules/unit_tests/plane_dupe_detector.dm:6]: Yeah then it spells air.plane

Comment (Time-Green) [reply code/modules/unit_tests/plane_dupe_detector.dm:6]: I think that’s pretty cool

Comment (Mothblocks) [reply code/modules/unit_tests/plane_dupe_detector.dm:6]: Bro

Comment (Time-Green) [reply code/modules/unit_tests/plane_dupe_detector.dm:6]: I’m really sad you don’t like it, but fine I’ll change it in a bit

Comment (Mothblocks) [reply code/modules/unit_tests/plane_dupe_detector.dm]: death by a thousand paper cuts/silly names

**Pull request merged**